Base/Python/slicer/util.py

```diff
@@ -474,7 +477,11 @@ def getNodes(pattern = "", scene=None):
       if (pattern == "" or
           fnmatch.fnmatchcase(name, pattern) or
           fnmatch.fnmatchcase(id, pattern)):
-        nodes[node.GetName()] = node
+        if useLists:
+          print("getNodes name: %s"%name)
```

Comment:
For logging use logging.debug (only written to log, not to the console), logging.info, etc. See https://www.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers/Style_Guide#Logging for details.